File: app/src/main/java/com/dumuzeyn/mp3player/MainActivity.java
package com.dumuzeyn.mp3player;

public class MainActivity extends MainActivityCore {
    @Override
    public void onRequestPermissionsResult(int requestCode, String[] permissions,
            int[] grantResults) {
        super.onRequestPermissionsResult(requestCode, permissions, grantResults);
        activityCoordinator.onRequestPermissionsResult(requestCode);
    }
}

File: app/src/main/java/com/dumuzeyn/mp3player/MainActivity.kt
package com.dumuzeyn.mp3player

open class MainActivity : MainActivityCore() {
    override fun onRequestPermissionsResult(
        requestCode: Int,
        permissions: Array<out String>,
        grantResults: IntArray,
    ) {
        super.onRequestPermissionsResult(requestCode, permissions, grantResults)
        activityCoordinator.onRequestPermissionsResult(requestCode)
    }
}
